route.go

```diff
@@ -81,7 +81,8 @@ type Route struct {
 type RouteInfo struct {
 	Name, Path, HandlerName string
 	// supported method of the route
-	Methods []string
+	Methods  []string
+	Handlers int
```

Comment:
叫 `HandlerNum` 更合适 😃 

```suggestion
	HandlerNum int
```